app/views/claims/claim_school.html.erb

```diff
@@ -1,9 +1,59 @@
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
+
+    <%= render("shared/error_summary", instance: current_claim) if current_claim.errors.any? %>
+
     <h1 class="govuk-heading-xl">
       Which school were you employed at between 6 April XXXX and 5 April XXXX?
     </h1>
 
-    <p class="govuk-body">Coming next...</p>
+    <% if @schools.blank? %>
+
+      <% if params[:school_search].present? %>
+        <p class="govuk-body">
+          <strong>No results match that search term. Try again.</strong>
+        </p>
+      <% else %>
+        <span id="school-search-hint" class="govuk-hint">
+          Search for the school name, use at least four characters.
+        </span>
+      <% end %>
+
+      <%= form_tag(claim_path("claim-school"), method: :get) do %>
+        <div class="govuk-form-group">
+          <%= label_tag :school_search, "School name", class: "govuk-label" %>
+          <%= text_field_tag :school_search, params[:school_search], class: "govuk-input" %>
+        </div>
+        <div class="govuk-form-group">
+          <%= submit_tag "Search", class: "govuk-button" %>
+        </div>
+      <% end %>
+
+    <% else %>
+
+      <span id="school-search-result-hint" class="govuk-hint">
+        Select your school from the search results.
+      </span>
+      <%= form_for current_claim, url: claim_path do |form| %>
+        <%= hidden_field_tag :school_search, params[:school_search] %>
+
+        <div class="govuk-form-group">
+          <fieldset class="govuk-fieldset" aria-describedby="school-search-result-hint">
+            <div class="govuk-radios">
+              <%= form.collection_radio_buttons :claim_school_id, @schools, :id, :name do |b| %>
+                <div class="govuk-radios__item">
+                  <%= b.radio_button( class: "govuk-radios__input") %>
```

Comment:
We could lose the perens here to make it consistent with the rest of the markup?

Reply:
👍